pkg/cli/mirror/copy_test.go

```diff
@@ -0,0 +1,372 @@
+			if err != nil {
+				t.Fatalf("The given path is not an OCI image : %v", err)
+			}
+			defer imgSrc.Close()
+			manifest, err := getManifest(context.TODO(), imgSrc)
+			if c.err != "" {
+				require.EqualError(t, err, c.err)
+			} else {
+				require.NoError(t, err)
+				require.Equal(t, c.layerCount, len(manifest.LayerInfos()))
+			}
+
+		})
+	}
+}
+
+func TestGetConfigPathFromLabel(t *testing.T) {
+	type spec struct {
+		desc            string
+		imagePath       string
+		configSha       string
+		expectedDirName string
+		err             string
+	}
+	cases := []spec{
+		{
+			desc:            "nominal case",
+			imagePath:       testdata,
+			configSha:       "sha256:c7c89df4a1f53d7e619080245c4784b6f5e6232fb71e98d981b89799ae578262",
+			expectedDirName: "/configs",
+			err:             "",
+		},
+		{
+			desc:            "sha doesnt exist fails",
+			imagePath:       testdata,
+			configSha:       "sha256:inexistingSha",
+			expectedDirName: "",
+			err:             "unable to read the config blob inexistingSha from the oci image: open testdata/artifacts/rhop-ctlg-oci/blobs/sha256/inexistingSha: no such file or directory",
+		},
+		{
+			desc:            "cfg layer json incorrect fails",
+			imagePath:       rotten_config,
+			configSha:       "sha256:c7c89df4a1f53d7e619080245c4784b6f5e6232fb71e98d981b89799ae578262",
+			expectedDirName: "",
+			err:             "problem unmarshaling config blob in c7c89df4a1f53d7e619080245c4784b6f5e6232fb71e98d981b89799ae578262: unexpected end of JSON input",
+		},
+		{
+			desc:            "label doesnt exist fails",
+			imagePath:       rotten_config,
+			configSha:       "sha256:c7c89df4a1f53d7e619080245c4784b6f5e6232fb71e98d981b89799ae5782ff",
+			expectedDirName: "",
+			err:             "label " + configsLabel + " not found in config blob c7c89df4a1f53d7e619080245c4784b6f5e6232fb71e98d981b89799ae5782ff",
+		},
+	}
+	for _, c := range cases {
+		t.Run(c.desc, func(t *testing.T) {
+			cfgDir, err := getConfigPathFromLabel(c.imagePath, c.configSha)
+			if c.err != "" {
+				require.EqualError(t, err, c.err)
+			} else {
+				require.NoError(t, err)
+				require.Equal(t, c.expectedDirName, cfgDir)
+			}
+
+		})
+	}
+}
+
+func TestFindFBCConfig(t *testing.T) {
+	type spec struct {
+		desc    string
+		options *MirrorOptions
+		err     string
+	}
+	cases := []spec{
+		{
+			desc: "nominal case",
+			options: &MirrorOptions{
+				From:             ociProtocol + testdata,
+				ToMirror:         "test.registry.io",
+				UseOCIFeature:    true,
+				OCIFeatureAction: OCIFeatureCopyAction,
+				OutputDir:        testdata,
+			},
+			err: "",
+		},
+		{
+			desc: "not a FBC image fails",
+			options: &MirrorOptions{
+				From:             ociProtocol + testdata,
+				ToMirror:         "test.registry.io",
+				UseOCIFeature:    true,
+				OCIFeatureAction: OCIFeatureCopyAction,
+				OutputDir:        "/tmp",
+			},
+			err: "unable to get OCI Image from /tmp: open /tmp/index.json: no such file or directory",
```

Comment:
And once again. Should be `oci:/tmp` apparently.

Reply:
Fixed
Thanks!